editor/js/Codemirror.js

```diff
@@ -0,0 +1,29 @@
+import { basicSetup } from 'codemirror';
+import { EditorView, keymap } from '@codemirror/view';
+import { javascript, esLint } from '@codemirror/lang-javascript';
+import { EditorState } from '@codemirror/state';
+import { oneDark } from '@codemirror/theme-one-dark';
+import { lintGutter, linter } from '@codemirror/lint';
+import { indentWithTab } from '@codemirror/commands';
+import Linter from 'eslint4b-prebuilt';
+
+const doc = '';
+export const Codemirror = ( { container, extensions = [] } ) => {
```

Comment:
Let's pass everything else as `...other`?? How do we enable themes, etc. in the future if we want to?

Reply:
Codemirror has different themes package, here in this file you can see one of these packages @codemirror/theme-one-dark. Also, it could have a white theme if we want to enable theming.